### PR TITLE
Add reference GitHub not commit rule

### PR DESCRIPTION
**Pull request by tslogan**
For #8

### CODE REVIEW

**Comment by tslogan** [comment]
Ready for review.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,6 +29,9 @@
       of the external developer.
     * Depending on the complexity of the pull request, discretion can be used to
       wait for multiple approvals before merging a pull request.
+* Issue and pull request numbers should only be referenced from within GitHub.
+  These issue numbers should not be referenced inside the body of a commit, as
+  it will start spamming the issue history.
```
Comment (amclain) [inline CONTRIBUTING.md]: How about changing this last line to this?

```
it may spam the issue history if certain Git operations are executed.
```

Comment (tslogan) [inline CONTRIBUTING.md]: Updated

**Comment by amclain** [comment]
:+1:

**Comment by tweatherford** [comment]
Like the grammar changes. :+1:  Approved

**Pull request merged**